src/algebra/indicator_function.lean

```diff
@@ -135,6 +135,30 @@ mul_indicator_eq_one.2 $ disjoint_empty _
 @[to_additive] lemma mul_indicator_empty' (f : α → M) : mul_indicator (∅ : set α) f = 1 :=
 mul_indicator_empty f
 
+lemma indicator_const_inverse_image {M} [has_zero M] (U : set α) (s : set M) (a : M) :
+  (U.indicator (λ x, a) ⁻¹' s) = set.univ ∨ (U.indicator (λ x, a) ⁻¹' s) = U ∨
+  (U.indicator (λ x, a) ⁻¹' s) = Uᶜ ∨ (U.indicator (λ x, a) ⁻¹' s) = ∅ :=
```

Comment:
Could be more compactly written `U.indicator (λ x, a) ⁻¹' s ∈ ({set.univ, U, Uᶜ, ∅} : set $ set α)` which is defeq to this.

Reply:
Thanks for the suggestion!  I went for it: I do not know whether there is a preferred spelling for this sort of thing, so I accepted your shorter, clearer formulation!

If anyone has a reason for preferring the `∨` galore, feel free to say so!